CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 # Change Log
 
+## v0.26.0
+
+Released on 2020-24-09
+
+### Major Changes
+
+* new: Address several sources of FPs, primarily from GKE environments. [[#1372](https://github.com/falcosecurity/falco/pull/1372)]
```

Comment:
This seems to be a rules update. Maybe the release note block of that PR needs to be changed.

Reply:
I don't understand what you are saying. What should I change what to? Do we have expectations about a pull request that changes rules? Can you please make a suggestion so that I understand what you are suggesting I do?

Reply:
I changed this to `rule:` instead of `new:` and moved the line to the third section

Reply:
I was referring to the [commit convetion](https://github.com/falcosecurity/falco/blob/master/CONTRIBUTING.md#commit-convention) for the release note block, as explained in the release.md.

Solved now! Thanks